src/database/queries/users.js

```diff
@@ -27,8 +34,37 @@ const checkId = (idNumber) => {
   return dbConnection.query(sql);
 };
 
+const checkMobile = (mobile) => {
+  const sql = {
+    text: 'SELECT mobile FROM users WHERE mobile=$1',
+    values: [mobile],
+  };
+  return dbConnection.query(sql);
+};
+
+const updateRole = (data) => {
+  const { role, userName } = data;
+  const sql = {
+    text: 'UPDATE users SET role=$1 where username =$2',
+    values: [role, userName],
+  };
+  return dbConnection.query(sql);
+};
+
+const deleteUser = (userName) => {
+  const sql = {
+    text: 'delete from users where username =$1',
```

Comment:
Don't forget to capitalize your keywords here, too.

Reply:
Done